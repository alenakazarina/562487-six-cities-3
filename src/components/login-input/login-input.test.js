import React from 'react';
import renderer from 'react-test-renderer';
import LoginInput from './login-input';

const mockFn = () => {};

describe(`LoginInput`, () => {
  it(`should render LoginInput email`, () => {
    const tree = renderer.create(
        <LoginInput
          name={`email`}
          value={`keks@gmail.com`}
          onChange={mockFn}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render LoginInput password`, () => {
    const tree = renderer.create(
        <LoginInput
          name={`password`}
          value={`SecretCatPass`}
          onChange={mockFn}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
