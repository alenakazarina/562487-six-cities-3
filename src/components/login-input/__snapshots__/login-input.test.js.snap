// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginInput should render LoginInput email 1`] = `
<div
  className="login__input-wrapper form__input-wrapper"
>
  <label
    className="visually-hidden"
  >
    Email
  </label>
  <input
    className="login__input form__input"
    name="email"
    onChange={[Function]}
    placeholder="Email"
    required={true}
    type="email"
    value="keks@gmail.com"
  />
</div>
`;

exports[`LoginInput should render LoginInput password 1`] = `
<div
  className="login__input-wrapper form__input-wrapper"
>
  <label
    className="visually-hidden"
  >
    Password
  </label>
  <input
    className="login__input form__input"
    name="password"
    onChange={[Function]}
    placeholder="Password"
    required={true}
    type="password"
    value="SecretCatPass"
  />
</div>
`;
